test/ring/middleware/test/anti_forgery.clj

```diff
@@ -128,7 +142,7 @@
   (is (thrown?
         AssertionError
         (wrap-anti-forgery (constantly {:status 200})
-                           {:error-handler (fn [request] {:status 500 :body "Handler"})
+                           {:error-handler  (fn [request] {:status 500 :body "Handler"})
```

Comment:
This is just a formatting change, and while correct, we can drop it from this PR, since it's not a relevant change.